urbanopt/geojson-modelica-translator
# feat: network and heating plant

#### Any background context you want to provide?
Part of improved GMT flexibility effort. We've been working towards representing a full district system.
#### What does this PR accomplish?
Implements the network and heating plant, and connects them to our existing systems.
#### How should this be manually tested?
Run the test `test_district_heating_district_new.py`. Inspect the resulting output, there should now be files in the `Plants` direcrtory, and the DistrictEnergySystem.mo should look right. Load it into Dymola and run the simulation.
#### What are the relevant tickets?
#230 
#### Screenshots (if appropriate)
Believe it or not, this is _after_ trying to reorganize the generated components to make it look pretty
<img width="571" alt="Screen Shot 2020-12-02 at 5 04 20 PM" src="https://user-images.githubusercontent.com/18518728/100947146-415f5a80-34c2-11eb-94e5-57e20345f3d2.png">
